cypress/e2e/inspectlet.spec.cy.ts

```diff
@@ -0,0 +1,28 @@
+describe(
+  'To test Inspectlet integration',
```

Comment:
Can be changed to `Inspectlet`

Reply:
Do want me to change the name of the file inspectlet.spec.cy.ts to Inspectlet.spec.cy.ts? If Yes, I haven't changed it to Inspectlet.spec.cy.ts because there is already a file in the smaller case `app.spec.cy.ts`, So I was just following that naming convention. Please let me know if it needs to be change.

Reply:
I'm referring to the suite name here - `describe(' To test...')`